src/js/component.js

```diff
@@ -1092,6 +1092,67 @@ class Component {
   }
 
   /**
+   * Get width or height of computed style
+   * @param  {String} widthOrHeight  'width' or 'height'
+   * @return {Number|Boolean} The bolean false if nothing was set
+   * @method currentDimension
+   */
+  currentDimension(widthOrHeight) {
+    let computedWidthOrHeight;
+
+    if (widthOrHeight !== 'width' && widthOrHeight !== 'height') {
+      log.warn('currentDimension only accepts width or height value');
+      return false;
+    }
+
+    if (typeof window.getComputedStyle !== 'undefined') {
+      const computedStyle = window.getComputedStyle(this.el_, null);
+      computedWidthOrHeight = computedStyle.getPropertyValue(widthOrHeight) || computedStyle[ widthOrHeight ];
+    }
+
+    if (this.el_.currentStyle) {
+      // ie 8 doesn't support computed style, shim it
+      // return clientWidth or clientHeight instead for better accuracy
+      const rule = 'client' + widthOrHeight.substr(0, 1).toUpperCase() + widthOrHeight.substr(1);
```

Comment:
would `offsetWidth`/`offsetHeight` be better?
Also, we have a utility for this called [`toTitleCase`](https://github.com/videojs/video.js/blob/cb3d70923736feea8dc3ef4087c4dd0420aae2dd/src/js/utils/to-title-case.js) (and it's already imported.
```js
const rule = `client${toTitleCase(widthOrHeight)}`
```